tests/integration/golang/fixtures/run.go

```diff
@@ -182,6 +182,15 @@ func (f RunFixtures) CreateTag(
 
 // CreateMetric creates a new test Metric.
 func (f RunFixtures) CreateMetric(ctx context.Context, metric *models.Metric) error {
+	if metric.ContextID == 0 {
```

Comment:
Again would prefer a semantic constant vs a magic number. Are the magic numbers the same?

Reply:
well '0' is the default value for an integer, so it's the equivalent of `if contextID == ""` or the like -- I think it's pretty common in go code, "no value yet given".

Reply:
can change to function...

Reply:
changed to `metric.Context.Json == nil` to match other tests